apidoc/Titanium/UI/Label.yml

```diff
@@ -166,8 +166,13 @@ properties:
     summary: Minimum font size when the font is sized based on the contents. 
     description: |
         Enables font scaling to fit and forces the label content to be limited to a single line.
+        
+        On Android, setting this property will force the `ellipsize` property to be
```

Comment:
sorry my bad, did review twice, first time added this comment. "lost" first review so inputted comments again then it looked like I had two reviews. Removed new and updated second review and this was left I guess from first time... :+1: